docs/rules/require-meta-has-suggestions.md

```diff
@@ -1,5 +1,7 @@
 # Require suggestable rules to implement a `meta.hasSuggestions` property (require-meta-has-suggestions)
 
+⚒️ The `--fix` option on the [command line](https://eslint.org/docs/user-guide/command-line-interface#-fix) can automatically fix some of the problems reported by this rule.
```

Comment:
```suggestion
⚒️ The `--fix` option on the [command line](https://eslint.org/docs/user-guide/command-line-interface#--fix) can automatically fix some of the problems reported by this rule.
```

Reply:
Need to fix the URL for all rule docs and the test: https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/pull/169

Reply:
👍